scripts/ench_books.zs

```diff
@@ -0,0 +1,37 @@
+
+// Efficiency 5
+assembler.recipeBuilder()
+    .inputs(<ore:paper> * 48)
+    .inputs(<ore:dustRedstone> * 16)
+    .outputs(<minecraft:enchanted_book>.withTag({StoredEnchantments: [{lvl: 5 as short, id: 32 as short}]}))
+    .duration(100)
+    .EUt(14)
+    .buildAndRegister();
+
+// Unbreaking 3
+assembler.recipeBuilder()
+    .inputs(<ore:paper> * 48)
+    .inputs(<ore:ingotInvar> * 16)
+    .outputs(<minecraft:enchanted_book>.withTag({StoredEnchantments: [{lvl: 3 as short, id: 34 as short}]}))
+    .duration(100)
+    .EUt(14)
```

Comment:
EUt at 14 is very weird, additionally, this book should definitely be MV. Use `60`.